pinot-common/src/main/java/org/apache/pinot/common/data/objects/JSONObject.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.pinot.common.data.objects;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.pinot.common.data.PinotObject;
+
+import com.beust.jcommander.internal.Lists;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+
+public class JSONObject implements PinotObject {
```

Comment:
The JSON library we use should already have a JSONObject? If so, we could use a facade?

Reply:
yes, we can change this to something that is fast and easy. This is just to show the concept. There are too many json libraries and found this link useful https://github.com/fabienrenaud/java-json-benchmark

Reply:
May be useful to rename this class as PinotJSONObject. Every JSON library (and there are several) probably has its own JSONObject :)

Reply:
will do